src/components/apps/terminal.js

```diff
@@ -237,16 +237,76 @@ export class Terminal extends Component {
                     result = "mkdir: missing operand";
                 }
                 break;
+            case "pwd":
+                let str = this.current_directory;  
+                result = str.replace("~","/home/vivek")
+                break;
             case "code":
-                if(words[0]==="."){
+                if(words[0]==="." || words.length === 0){
                     this.props.openApp("vscode");
                 } else {
-                    result = "Command '" + main + "' not found, or not yet implemented.<br>Available Commands: [ cd, ls, echo, clear, exit, mkdir,code]"; 
+                    result = "Command '" + main + "' not found, or not yet implemented.<br>Available Commands:[ cd, ls, pwd, echo, clear, exit, mkdir, openAllapps- code, spotify, chrome, about-vivek, todoist, trash, settings, sendmsg]"; 
```

Comment:
just remove 'openAllapps- ' text, everything else is fine

Reply:
done!